arches/app/etl_modules/branch_csv_importer.py

```diff
@@ -116,6 +117,11 @@ def create_tile_value(self, cell_values, data_node_lookup, node_lookup, row_deta
                     config["nodeid"] = nodeid
                 except TypeError:
                     config = {}
+                if datatype == "string":
+                    try:
+                        source_value = ast.literal_eval(source_value)
```

Comment:
This check actually belongs in the string datatype's transform_value_for_tile method. If it's moved into transform_value_for_tile, then this same block can be removed from `import_single_csv` as well.

Reply:
Moved, thanks.